Tests/Unit/NetworkServer/ModuleConnectionHostTest.cs

```diff
@@ -115,6 +114,48 @@ public async Task On_Desired_Properties_Incorrect_Update_Should_Not_Update(strin
             Assert.Equal(facadeCode, localLoRaDeviceApiServiceBase.AuthCode);
         }
 
+        [Theory]
+        [InlineData(-100)]
+        [InlineData(0)]
+        [InlineData(5)]
+        [InlineData(400)]
+        [InlineData(1000)]
+        public async Task On_Desired_Properties_Correct_Update_Should_Update_Processing_Delay(int processingDelay)
+        {
+            var networkServerConfiguration = new NetworkServerConfiguration();
+            var classCMessageSender = Mock.Of<IClassCDeviceMessageSender>();
+            var loRaDeviceRegistry = Mock.Of<ILoRaDeviceRegistry>();
+            var loRaModuleClientFactory = Mock.Of<ILoRaModuleClientFactory>();
+
+            await using var moduleClient = new ModuleConnectionHost(networkServerConfiguration, classCMessageSender, loRaModuleClientFactory, loRaDeviceRegistry, loRaDeviceApiServiceBase, NullLogger<ModuleConnectionHost>.Instance, TestMeter.Instance);
+
+            Assert.Equal(400, networkServerConfiguration.ProcessingDelayInMilliseconds);
```

Comment:
[nit] Did you consider making 400 a constant so that if we change the value we wouldn't need to fix these tests?

Reply:
Done in df7188bbce978d98caceb7410658ed48769a7b43